test/tvla_test.py

```diff
@@ -17,3 +19,24 @@ def test_help():
     tvla = TvlaCmd(Args('--help')).run()
     # Assert that a message is printed on stdout or stderr.
     assert(len(tvla.stdout()) != 0 or len(tvla.stderr()) != 0)
+
+
+def ttest_significant(ttest_trace) -> bool:
+    """Determine if a t-test trace contains a significant deviation from the mean."""
+    threshold = 4.5
+    abs_max = np.max(np.abs(ttest_trace), axis=3)
+    return np.any(abs_max > threshold)
+
+
+def test_general_leaking_histogram():
+    hist_path = TestDataPath('tvla_general/sha3_hist_leaking.npz')
+    tvla = TvlaCmd(Args(f'-g -i {hist_path}')).run()
```

Comment:
Nit: add the `-m sha3` argument. It's not strictly required at the moment though.

Reply:
Thanks :+1: changed

Reply:
Excellent, thanks Andreas!